cruiser/scenarios/eg01_eg23.py

```diff
@@ -18,6 +18,15 @@ def startmonth(self, value):
     def mine_throughput(self, value):
         self.sim['simulation']['facility'][0]['config']['Source']['throughput'] = value
 
+    @inparam(default=.003, widget='FloatSlider', min=0.0, max=0.00711, step=0.0001, readout_format='.4f')
+    def enrichment_tails_assay(self, value):
+        self.sim['simulation']['facility'][1]['config']['Enrichment']['tails_assay'] = value
+
+    @inparam(default=1e100, widget='FloatSlider', min=0.0, max=10, step=0.1, readout_format='.4f')
+    def new_parameter(self, value):
```

Comment:
`new_parameter` is not a good name. Please change to something more meaningful, like `mixer_throughput`.